### ISSUE TITLE
Centroid action missing in api docs

### ISSUE THREAD

**Issue by nilsnolde** [valhalla/valhalla#3226]
just realized that there's no API docs for the centroid service (introduced in #2734 ). was that on purpose @kevinkreiser (experimental like /expansion so far)? 

just had the task to choose a place and hotel for our company event and wanted to nerd it out a little haha (don't need the docs really, would've saved some time though)

**Comment by kevinkreiser** [comment]
Yeah it was kind of on purpose. Similar to the expansion endpoint. I've taken to this idea that it's OK to add new APIs undocumented until they feel sufficiently bug free so as to want to offer regular support of them. A better option might be document them and slap a big label of beta on them? I also don't like to release a non-trivial top level api without a properly working demo which this one and expansion both have though I swore there were still some bugs in the centroid one

**Comment by nilsnolde** [comment]
> and slap a big label of beta on them

IMO that's more productive, since no one necessarily expects a stable bug-free service then, but there'd still be enough folks who like being experimental. also helps with bug reporting no? not sure about you, but I _actually_ play surprisingly little around with valhalla services, I'm not a good candidate to discover runtime bugs unless I unleash it on some users 😄
